spec/operators/defaultIfEmpty-spec.js

```diff
@@ -1,50 +1,42 @@
-/* globals describe, it, expect */
+/* globals describe, it, expect, expectObservable, hot */
 var Rx = require('../../dist/cjs/Rx');
 var Observable = Rx.Observable;
 
 describe('Observable.prototype.defaultIfEmpty()', function () {
-  it('should return the argument if Observable is empty', function (done) {
-    var emptyObservable = Observable.empty();
-    emptyObservable.defaultIfEmpty(2)
-      .subscribe(function (x) {
-        expect(x).toBe(2);
-      }, null, done);
+  var defaultValue = 'x';
```

Comment:
The reasoning behind keeping repetition is: we should be able to copy and paste a test case and it should contain all information needed without context surrounding it.